<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SRT_edit_form</class>
 <widget class="QMainWindow" name="SRT_edit_form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Effect_combo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Benchmark_combo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>video</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="FileName_text">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>371</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Font_combo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Pt_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>189</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>pt</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>輸出之檔案名稱(Video Name)：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Return_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一頁</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Finish_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>結束</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Relative_combo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bottom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>center</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>top</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>324</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>輸出路徑(path)：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="FontSpace_combo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>235</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Small</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Middle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Large</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="SrtName_text">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Embed_btn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>嵌入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>生成的字幕檔案(SRT)：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Of_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>275</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>of</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>189</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>字型大小(Font Size)：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>279</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>字幕位置(Subtitle Location)：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>105</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>嵌入字幕(Embed Font)：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>145</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>字幕特效(Effect)：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Location_combo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>above</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>字形間距(Font Spacing)：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="Path_text">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>325</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FontSize_line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>195</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Folder_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>55</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>選擇並編輯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PathFind_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>選擇</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
